lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1013,7 +1014,7 @@ void HdVP2Mesh::_UpdateDrawItem(
                 // Use fallback shader if there is no material binding or we
                 // failed to create a shader instance from the material.
                 if (!stateToCommit._shader) {
-                    const GfVec3f& color = colorArray[0];
+                    const GfVec3f& color = UsdMayaColorSpace::ConvertLinearToMaya(colorArray[0]);
```

Comment:
@HamedSabri-adsk Do you think this is a good idea? Should I extend it to the other supported geometries?